airbyte-ci/connectors/pipelines/README.md

```diff
@@ -555,6 +557,30 @@ Migrate connectors the poetry package manager.
 Migrate source-openweather to use the base image:
 `airbyte-ci connectors --name=source-openweather migrate-to-poetry`
 
+### <a id="connectors-migrate_to_inline_schemas"></a>`connectors migrate_to_inline_schemas` command
+
+Migrate .json schemas into manifest.yaml files, when present.
```

Comment:
```suggestion
Migrate `.json` schemas into `manifest.yaml` files, when present.

Caveat: that might break if the json schemas used `$params`.
```

Reply:
The $parameters actually seem to work in newer CDK versions. The ones it was breaking before were quite old.